### PR TITLE
Update MathML font documentation

### PR DESCRIPTION
**Pull request by fred-wang**
#### Summary
- Make the instructions less Mozilla-specific so that it can be used for
  other browsers too.
- Update installation instructions to take into account recent changes
  in macOS, Linux and Android.
- Remove instructions for Firefox OS.
- Remove "Using mathematical fonts on Web pages" section, as it's more
  targetted to authors than users.
- Delete unused test page to check whether math fonts are installed.

#### Motivation

- Update installation instructions to take into account recent changes
  in macOS, Linux and Android.
- Make the instructions less Mozilla-specific so that it can be used for
  other browsers too.

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by fred-wang** [comment]
cc @sideshowbarker

**Comment by teoli2003** [comment]
Have you use `yarn content delete <slug> --redirect <newslug>` to delete the page here? This takes care of cleaning the wiki history file, and a few other cleanings.

If so, all fine. If not, could you do it, it is the most efficient way to delete a page.

**Comment by fred-wang** [comment]
@teoli2003 This is the command I used (`--recursive` was probably not needed):

      yarn content delete Web/MathML/Fonts/Test --recursive --redirect Web/MathML/Fonts

File: files/en-us/web/mathml/fonts/index.md [hunk 1]
```diff
@@ -1,75 +1,84 @@
...
   - Fonts
   - MathML
   - Project
 ---
 Fonts with appropriate Unicode coverage and Open Font Format features are required for good math rendering.
-This page describes how users can install and use such math fonts with Mozilla's MathML engine.
-Note that most of these instructions may as well apply to other Web rendering engines.
+This page describes how users can install such math fonts to properly display MathML in browsers.
 
 ## Installation Instructions
```
Comment (teoli2003) [inline files/en-us/web/mathml/fonts/index.md:12]: We use sentence casing in headings:
```suggestion
## Installation instructions
```

File: files/en-us/web/mathml/fonts/index.md [hunk 2]
```diff
@@ -79,45 +88,49 @@ However, you might need to ensure that the fonts are known by your system. Typic
...
+> **Note:** Noto Sans Math provides good Unicode coverage for math symbols and [Google plans to add support for math layout features](https://github.com/notofonts/math/issues/14#issuecomment-1161414446).
 
 ### Other systems
 
-On other systems, consider installing a [MATH font](#fonts_with_a_math_table) using your package manager. Note that these fonts are generally delivered with TeX distributions such as [TeX Live](https://www.tug.org/texlive/), but you might need to follow specific instructions so that your system is aware of the fonts. As a last resort, install the [MathML fonts add-on](https://addons.mozilla.org/firefox/addon/mathml-fonts/).
+On other systems, consider installing a [font with a MATH table](#fonts_with_a_math_table) using your package manager. Note that these fonts are generally delivered with TeX distributions such as [TeX Live](https://www.tug.org/texlive/), but you might need to follow specific instructions so that your system is aware of the fonts. As a last resort, install the [MathML fonts add-on](https://addons.mozilla.org/firefox/addon/mathml-fonts/).
 
 ## Advanced setup
 
 ### Arabic Mathematical Alphabetic Symbols
```
Comment (teoli2003) [inline files/en-us/web/mathml/fonts/index.md:116]:
```suggestion
### Arabic mathematical alphabetic symbols
```

File: files/en-us/web/mathml/fonts/index.md [hunk 3]
```diff
@@ -79,45 +88,49 @@ However, you might need to ensure that the fonts are known by your system. Typic
...
 
-Currently, very few fonts have appropriate glyphs for the Arabic Mathematical Alphabetic Symbols. If you are likely to need these characters, we recommend to install the XITS or [Amiri](https://www.amirifont.org/) fonts.
+Currently, very few fonts have appropriate glyphs for the Arabic Mathematical Alphabetic Symbols. If you are likely to need these characters, we recommend to install the _XITS_ or [Amiri](https://www.amirifont.org/) fonts.
 
 ### Installation without Administrator Privilege
```
Comment (teoli2003) [inline files/en-us/web/mathml/fonts/index.md:120]:
```suggestion
### Installation without administrator privilege
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few typographic improvements (normally I'll just commit them, but I think, as you are updating more pages, it is useful you know about some conventions).

Thank you very much, btw!

**Comment by fred-wang** [comment]
@teoli2003 Thanks for the review. I believe your feedback is addressed by the latest commits.

**Review by teoli2003 — APPROVED** [review]
They are! Thank you!

**Pull request merged**